### PR TITLE
Added type for tabs click events

### PR DESCRIPTION
**Pull request by mitchelldemler**

### CODE REVIEW

File: components/tabs/index.tsx [hunk 1]
```diff
@@ -17,8 +17,8 @@ export interface TabsProps {
   hideAdd?: boolean;
   onChange?: (activeKey: string) => void;
   onTabClick?: Function;
-  onPrevClick?: (e) => void;
-  onNextClick?: (e) => void;
+  onPrevClick?: (e: React.MouseEventHandler<any>) => void;
+  onNextClick?: (e: React.MouseEventHandler<any>) => void;
```
Comment (yesmeck) [inline components/tabs/index.tsx]: It should be `onNextClick?: React.MouseEventHandler<any>`?

Comment (afc163) [inline components/tabs/index.tsx]: oops

Comment (mitchelldemler) [inline components/tabs/index.tsx]: oops. #5621

Comment (mitchelldemler) [inline components/tabs/index.tsx]: If possible could noImplicitAny be added to your tsconfig during the CI build?

**Pull request merged**